src/index.html

```diff
@@ -2,6 +2,8 @@
 <html lang="en">
 
 <head>
+  <script defer data-domain="preview.micronutrient.support"
```

Comment:
Is this needed if it is also created in the ngOnInit of the app-component?

Reply:
oof, yeah that should be there any more. Let me recheck and will tag you when good to go